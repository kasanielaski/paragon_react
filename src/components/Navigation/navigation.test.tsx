import { describe, it, expect } from 'vitest'
import { render } from '@testing-library/react';

import { Navigation } from '../index';

const props = {
  page: 1,
  isLastPage: true,
  handlePageChange: () => {}
};

describe('Navigation component', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(<Navigation {...props} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
