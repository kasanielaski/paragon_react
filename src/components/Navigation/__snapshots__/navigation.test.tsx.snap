// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Navigation component > should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-jbAkgO gpPZIG"
  >
    <button
      disabled=""
    >
      Previous
    </button>
    <button
      disabled=""
    >
      Next
    </button>
  </div>
</DocumentFragment>
`;
